server/render.js

```diff
@@ -26,8 +27,8 @@ export async function render (url, ctx = {}, {
     errorComponent
   ] = await Promise.all([
     Component.getInitialProps ? Component.getInitialProps(ctx) : {},
-    read(join(dir, '.next', 'bundles', 'pages', path)),
-    read(join(dir, '.next', 'bundles', 'pages', dev ? '_error-debug' : '_error'))
+    read(join(pages, path)),
```

Comment:
We can't use pages here. It's the bundles directory here. On the top it's pages directory.
It's nice to move common path into a variable. (In that case, it would be the .next dir path)